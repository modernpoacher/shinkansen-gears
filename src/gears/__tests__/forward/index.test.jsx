import React from 'react'
import renderer from 'react-test-renderer'

import Forward from '#gears/forward'

jest.mock('react-router-dom')

describe('#gears/forward', () => {
  it('renders', () => {
    const component = (
      <Forward pathname='MOCK PATHNAME' />
    )

    return expect(renderer.create(component).toJSON())
      .toMatchSnapshot()
  })
})
