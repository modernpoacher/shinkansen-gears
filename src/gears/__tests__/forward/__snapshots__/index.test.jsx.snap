// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#gears/forward renders 1`] = `
<li
  className="shinkansen-gears-forward"
/>
`;
